react-frontend/src/listings/EditListing.js

```diff
@@ -158,7 +177,43 @@ function EditListing() {
   return (
     <div className="vertical-center margin">
       <div className="small-container drop-shadow">
-        <h2>Edit Listing</h2>
+      <div className="vertical-center" style={{ display: 'flex', alignItems: 'center', marginRight: "1.5em" }}>
+  <button
+    onClick={handleCancel}
+    style={{
+      background: 'grey',
+      border: 'none',
+      cursor: 'pointer',
+      marginRight: '1em',
+      padding: '0',
```

Comment:
you theoretically shouldn't have to set padding to 0, border to none. remove these lines unless any other css is affecting it

Reply:
I thought so too, but apparently I do need it. However, I'll be replacing this monstrosity with a more simple arrow next sprint